README.md

```diff
@@ -102,6 +102,10 @@ let left = divideLeftMultiplyRight.apply(.Left(4)) // 2
 let right = divideLeftMultiplyRight.apply(.Right(7)) // 14
 ``` 
 
+**Operators**
+
+See [Operators](https://github.com/typelift/Operadics) for a list of supported operators.
```

Comment:
Link to https://github.com/typelift/Operadics#operators to have it jump to the table